modules/claims_api/app/controllers/claims_api/v2/veterans/evidence_waiver_controller.rb

```diff
@@ -9,13 +9,42 @@ module ClaimsApi
   module V2
     module Veterans
       class EvidenceWaiverController < ClaimsApi::V2::ApplicationController
+        before_action :set_lighthouse_claim
+        before_action :set_bgs_claim!
+        before_action :validate_sponsor!
+        before_action :validate_file_number!
+        before_action :verify_if_dependent_claim!
```

Comment:
woops, technically these 3... apologies, :14, :15 and :16

Reply:
@FonzMP The dependent claim check is what determines if this is filed by a dependent or not.  That is where the log out occurs. 

The sponsor and file number I get as those were just removed and need to be cleaned out here, but that third one, unless I am missing something on this, needs to be there and handles of some of the ACs the ticket, right?

Reply:
woops, thought this was to confirm the old sponsorIcn, you're right.

Reply:
so just :14, :15 are still relevant for the above (removal) request, right?